AppController/djinn.rb

```diff
@@ -1260,22 +1250,22 @@ def get_app_upload_status(reservation_id, secret)
   #   secret: A string with the shared key for authentication.
   # Returns:
   #   A JSON string with the statistics of the nodes.
-  def get_stats_json(secret)
+  def get_cluster_stats_json(secret)
```

Comment:
This specific function does have only one form and it is JSON. But other functions of AppContoller not always uses JSON. So I'd say it's sensible to leave `_json` suffix.